app/code/core/Mage/Newsletter/Model/Resource/Queue/Collection.php

```diff
@@ -43,10 +43,6 @@ protected function _construct()
     }
 
     /**
-     * Joines templates information
-     *
-     * @deprecated since 1.4.0.1
-     *
```

Comment:
How come you just deleted the deprecated message but not the method?

Reply:
yep that module is used in a couple of place and I didn't see a replacement

<img width="1088" alt="Screenshot 2025-02-25 alle 18 46 43" src="https://github.com/user-attachments/assets/c205e186-ddeb-4e2b-8a21-0fcea181145b" />